### DISCUSSION TITLE
Debuggin Sync failure

### DISCUSSION THREAD

**Discussion by tpireaux** [romanz/electrs#578] [Q&A] [answered]
Following a [guide](https://stadicus.github.io/RaspiBolt/raspibolt_50_electrs.html) to get electrs up and running on a pi4. Running into two errors that I am trying to track down solutions for. Any tips? Thanks

electrs: 0.9.1
bitcoin core: 0.21.1

electrs.conf RPC user/pass (tried both):

- > cookie = "user:pass"
  - fails with error 1
- > auth = "user:pass"
  - eventually fails with error 2
  - when I check out that block using `bitcoin-cli getblock hash` it appears to be fine

Error 1

> Error : electrs failed
> 
> Caused by:
>     0: failed to open bitcoind cookie file: /mnt/ext/bitcoin/.cookie
>     1: No such file or directory (os error 2)

Error 2

> Error: electrs failed
> 
> Caused by:
>     0: sync failed
>     1: failed to get block 000000000000006457478825e0608a93df85da294d623b80c85e4eb352085672
>     2: receiving on an empty and disconnected channel

**Comment by Kixunil** [comment, marked as answer]
Please see [this](https://github.com/romanz/electrs/#usage):

> External guides such as RaspiBolt can be out-of-date and have various problems. At least double-check that the guide you're using is actively maintained. If you can't use our guide ask about what you don't understand or consider using automated deployments.

**Reply by tpireaux** [reply]
In  **bitcoin.conf** adding `whitelist=download@127.0.0.1` fixed my random sync problems. Thank you.
